Source/Compiler.m

```diff
@@ -61,7 +62,7 @@ + (NSString *)unique:(NSString *)identifier
 - (NSMutableString *)getTemplateWithName:(const char *)name
 {
 	size_t length;
-	const void *data = getsectdata ("__DATA", name, &length);
+	const void *data = getsectiondata (&_mh_execute_header, "__DATA", name, &length);
```

Comment:
Der generierte Code für die Tests sah hinterger identisch aus. Im Netz haben alle den `_mh_execute_header` genutzt. `NULL` schien aber auch zu funktionieren. So 100% hab ich nicht verstanden, was hier abgeht. 
Eventuell kann hier @hydrixos sagen, ob das okay ist so?

Reply:
Schaut für mich richtig aus!

Das Problem ist, dass das pegged-Binary kein Bundle ist. Daher werden hier ein paar Daten als Binär-Daten-Sektion in das Executable mit reingebacken und auf diesem Weg dann wieder ausgelesen.